packages/shared/src/components/modals/report/ReportPostModal.tsx

```diff
@@ -164,7 +164,7 @@ export function ReportPostModal({
       onReport={onReportPost}
       reasons={reportOptionsForActiveReason}
       heading="Report post"
-      title={`"${post?.title}"`}
+      title={post?.title}
```

Comment:
Not sure if we wanted the title "quoted", maybe this preserves it?
```suggestion
      title={post?.title ? `"${post.title}"` : undefined}
```

Reply:
Ah yes didn't think of that 🤦‍♀️ thanks for noticing

Reply:
Also in some places we use `DEFAULT_POST_TITLE` not sure if that was intended here maybe as well?